route/build/src/mpi_utils.f90

```diff
@@ -9,6 +9,7 @@ MODULE mpi_utils
   USE globalData, ONLY: masterproc
   USE public_var, ONLY: root
   USE public_var, ONLY: iulog
+  USE shr_sys_mod,ONLY: shr_sys_flush
```

Comment:
Hi Erik, wondering how this works for stand-alone. does stand-alone access to shr_sys_mod?

Reply:
We can make it available. I think the standalone build will have to change a little bit, but not too much. I plan to make a crack at that a bit later.